### DISCUSSION TITLE
How can we update `Array` Column type ?

### DISCUSSION THREAD

**Discussion by sumitsharansatsangi** [piccolo-orm/piccolo#589] [Q&A] [answered]
I tried to update the `Array` column type variable but didn't succeed.

**Code :**
```python
await User.update({
            User.phone: User.phone.append(new_phone)
        }, )
````

**Error Received :**

```
'Array' object has no attribute 'append'
```

Tried following also but doesn't work 

**Code :**
```python
await User.update({
            User.phone: User.phone + [new_phone]
        }, )
````

**Error Received :**

```
unsupported operand type(s) for +: 'Array' and 'list'
```

**Comment by dantownsend** [comment, marked as answer]
Yeah, this isn't supported currently, but I've been able to add it. Will do a PR.

**Reply by dantownsend** [reply]
Implemented here: https://github.com/piccolo-orm/piccolo/pull/591
